### PR TITLE
feat: changes for new common job-manager umbrella and new build and push (MAPCO-4080)

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: helm/templates/_tplValues.tpl [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+End of usage example
+*/}}
+
+{{/*
+Custom definitions
+*/}}
+
+{{- define "common.db.merged" -}}
+{{- include "common.tplvalues.merge" ( dict "values" ( list .Values.db .Values.global.db ) "context" . ) }}
+{{- end -}}
```
Comment (asafMasa) [inline helm/templates/_tplValues.tpl:50]: please check EOF in all files

File: helm/values.yaml [hunk 2]
```diff
@@ -3,6 +3,19 @@ global:
   tracing: {}
   metrics: {}
   environment: {}
+  ingress:
+    domain: 'apps.aroapp.io'  
+  db:
+    host: "localhost"
+    name: "common"
```
Comment (asafMasa) [inline helm/values.yaml:10]: keep empty to be overrided from shared-deployments

File: helm/values.yaml [hunk 3]
```diff
@@ -30,16 +43,17 @@ local:
...
-    secrets:
-      useExternal: false
-      externalSecretName: ''
-
+db:
+  host: "localhost"
+  name: "common"
+  port: 5432
+  schema: "HeartbeatManager"
+  sslEnabled: false
```
Comment (asafMasa) [inline helm/values.yaml:51]: keep empty to be overrided from shared-deployments

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/output [hunk 4]
```diff
@@ -0,0 +1,121 @@
+---
```
[the remark this answers is not in the document]
Comment (almog8k) [reply helm/output:1]: deleted

**Review by asafMasa — APPROVED** [review]

**Review by netanelC — CHANGES_REQUESTED** [review]
1. In workflows, remove the whitespaces from the last line if the files.
2. Remove the _resources.tpl

**Review by netanelC — APPROVED** [review]

**Pull request merged**